tools/rw-heatmaps/README.md

```diff
@@ -12,15 +12,35 @@ To get a mixed read/write performance evaluation result:
 ```
 `rw-benchmark.sh` will automatically use the etcd binary compiled under `etcd/bin/` directory.
 
-Note: the result csv file will be saved to current working directory. The working directory is where etcd database is saved. The working directory is designed for scenarios where a different mounted disk is preferred.
+Note: the result CSV file will be saved to current working directory. The working directory is where etcd database is saved. The working directory is designed for scenarios where a different mounted disk is preferred.
 
 ### Plot Graphs
-To generate two images (read and write) based on the benchmark result csv file:
+To generate two images (read and write) based on the benchmark result CSV file:
 ```sh
 # to generate a pair of read & write images from one data csv file
-./plot_data.py ${FIRST_CSV_FILE} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
+go run . ${CSV_FILE} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
+```
+
+To generate two images (read and write) showing the performance difference from two result CSV files:
+```sh
+# to generate a pair of read & write images from one data csv file
+go run . ${CSV_FILE1} ${CSV_FILE2} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
```

Comment:
```suggestion
rw-heatmaps ${CSV_FILE1} ${CSV_FILE2} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
```

Reply:
Please also refer to https://github.com/etcd-io/etcd/blob/main/tools/benchmark/README.md to add installation and usage guide.

Reply:
Done